### PR TITLE
Bump version to 0.5.0

### PR DESCRIPTION
**Pull request by thewtex**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by thewtex** [comment]
@conda-forge-admin, please rerender

**Comment by thewtex** [comment]
@conda-forge-admin, please rerender

**Comment by thewtex** [comment]
@ctrueden @hadim osx is failing to run the tests with:

> No Java runtime present, requesting install.

Any ideas?

**Comment by imagejan** [comment]
> No Java runtime present, requesting install.

Maybe we need to add `openjdk=8` to the requirements in `meta.yml`?
https://github.com/conda-forge/pyimagej-feedstock/blob/1036740cd49bfeb4c58491472c9596f5a851ac12/recipe/meta.yaml#L17-L25

Do you think this could solve it?

**Comment by thewtex** [comment]
@imagejan good idea -- added in d0e9a26 , but it did not address the issue :-(.

Tried another work around suggested by @ctrueden

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ requirements:
 test:
   imports:
     - imagej
+    - skip: True  # [osx]
```
Comment (imagejan) [inline recipe/meta.yaml]: Doesn't this need to be one level up, in the `test:` section, not under `imports`?

Comment (thewtex) [reply recipe/meta.yaml]: @imagejan -- good catch!

**Comment by ctrueden** [comment]
Thanks so much @thewtex for working on this!

**Pull request merged**